src/dist/replication/lib/replica_2pc.cpp

```diff
@@ -44,6 +44,16 @@ void replica::on_client_write(dsn::message_ex *request, bool ignore_throttling)
         return;
     }
 
+    if (_stub->_max_allowed_write_size &&
+        request->body_size() > _stub->_max_allowed_write_size) {
+        dwarn_replica("client from {} write request body size exceed threshold = {}, it will be "
```

Comment:
Is it able to logging more info, like gpid. Of cource, logging key would be more helpful, but it seems impossible here.

Reply:
1. The log refer to “abnormal get” log. I think it not more than later.
2.The response will result in client exception, we can get the more exception such as key (https://github.com/XiaoMi/pegasus-java-client/issues/92).